massa-protocol-exports/src/error.rs

```diff
@@ -55,13 +55,13 @@ pub enum ProtocolError {
 
 #[derive(Debug)]
 pub enum HandshakeErrorType {
-    HandshakeIdAlreadyExistError(String),
-    HandshakeTimeoutError,
-    HandshakeInterruptionError(String),
-    HandshakeWrongMessageError,
-    HandshakeKeyError,
-    HandshakeInvalidSignatureError,
-    IncompatibleVersionError,
+    HandshakeIdAlreadyExist(String),
+    HandshakeTimeout,
+    HandshakeInterruption(String),
+    HandshakeWrongMessage,
+    HandshakeKey,
+    HandshakeInvalidSignature,
+    IncompatibleVersion,
 }
 
```

Comment:
This can totally be removed. It's unused! Remove also his reference upper in ProtocolError :)

Reply:
That's weird ... I'll look into that

Reply:
That was duplicated during the split of protocol and network, without checking what variants were used where